src/main/java/codeu/model/store/basic/DefaultDataStore.java

```diff
@@ -95,19 +99,35 @@ public boolean isValid() {
     return messages;
   }
 
+  public List<Tag> getAllTags() {
+    return tags;
+  }
+
   private void addRandomUsers() {
 
     List<String> randomUsernames = getRandomUsernames();
     Collections.shuffle(randomUsernames);
 
-    for (int i = 0; i < DEFAULT_USER_COUNT; i++) {
+    for (int i = 0; i < DEFAULT_MESSAGE_COUNT; i++) {
```

Comment:
why is this changed from User to Message counts?

Reply:
Good catch, I had meant to use message count for addRandomTags and not addRandomUsers